src/python/pants/backend/python/target_types_rules.py

```diff
@@ -484,7 +484,7 @@ async def validate_python_dependencies(
     # ... should be replaced with calls to InterpreterConstraintsField.value_or_global_default.
     deprecated_conditional(
         lambda: bool(non_subset_items),
-        removal_version="2.13.0.dev2",
+        removal_version="2.13.0.dev3",
```

Comment:
I looked at this a bit - too much going on there for what I need to get done here in a short bit before disappearing.